backend/src/v5/schemas/tickets/validators.js

```diff
@@ -53,7 +53,22 @@ Validators.generateViewValidator = (isUpdate, required, isComment) => {
 		[viewGroups.TRANSFORMED]: generateGroupArraySchema({
 			transformation: Yup.array().of(Yup.number()).length(16).required(),
 		}),
-	}).default(undefined), true);
+		[viewGroups.SELECTED]: generateGroupArraySchema({
+			color: types.color3Arr,
+			opacity: Yup.number().max(1).test('opacity value', 'Opacity value must be bigger than 0', (val) => val === undefined || val > 0),
```

Comment:
selected shouldn't have an opacity value, as we're not doing colour override (sorry should've made that clear!)